app/models/comment.rb

```diff
@@ -1,10 +1,13 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
-  belongs_to :post
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] [optional] @rebel216 great work working on the changes requested, however, am still having this error from your app when I run migrations : 
![image](https://user-images.githubusercontent.com/61408860/204623519-15453bfd-582c-4cc0-9166-e8bb1ae44efc.png)

Reply:
ca nu be please more elaborative about exactly what's missing in the readme ?? Because I did used that template .

Reply:
it is done in developer branch.